components/SearchBar.tsx

```diff
@@ -0,0 +1,23 @@
+import { TextInput } from '@primer/react'
+import { Dispatch, SetStateAction, FC } from 'react'
+import { SetterOrUpdater } from 'recoil'
+import { SearchIcon } from '@primer/octicons-react'
+
+const SearchBar: FC<{
+    searchValue: string
+    setSearchValue: SetterOrUpdater<string> | Dispatch<SetStateAction<string>>
+}> = ({ searchValue, setSearchValue }) => {
+    return (
+        <TextInput
+            icon={SearchIcon}
+            className='border py-1 rounded-lg searchbox-candymachine f5 pl-2'
+            type='search'
+            placeholder='Search candy machine'
```

Comment:
As the component is generic, I think that the text of the placeholder should be a prop

Reply:
You are right. I didn't think about that

Reply:
This needs to be resolved